### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by santhoshpemmaka**
adding: a single product page, profile page, and signup feature to the E-commerce website.

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
great work . Nothing suggestion from my side . You can proceed for merge

File: src/components/index.js [hunk 1]
```diff
@@ -17,4 +24,10 @@ export {
 	Login,
 	SignUp,
 	ErrorPage,
+	ProfilePage,
```
Comment (VarshithSSV) [inline src/components/index.js:27]: I like export components one index file

File: src/components/SignUp/SignUp.jsx [hunk 2]
```diff
@@ -13,7 +21,50 @@ const SignUp = () => {
...
+
+	const usersignupHandler = async (dispatch, signupDeatils) => {
+		try {
+			const response = await axios.post("/api/auth/signup", {
+				email: signupDeatils.emailName,
+				passWord: signupDeatils.passWord,
+				firstName: signupDeatils.firstName,
+				lastName: signupDeatils.lastName,
+			});
+			if (response.status === 200 || response.status === 201) {
```
Comment (VarshithSSV) [inline src/components/SignUp/SignUp.jsx:38]: I like the checking api response status and update reducer is good approach.

**Review by VarshithSSV — APPROVED** [review]

**Comment by VarshithSSV** [comment]
Code looks good and code structure also good. Awesome Work 💯 .

Good to merge.

**Pull request merged**